haskell/src/Monocle/Env.hs

```diff
@@ -25,7 +25,8 @@ data Env = Env
 -- | 'Env' is the global environment
 data AppEnv = AppEnv
   { config :: IO (Bool, Config.Config),
-    aEnv :: Env
+    aEnv :: Env,
+    cRStatus :: TVar [(Text, Bool)]
```

Comment:
nit: i'm not sure what `CRStatus` means, could we find a more explicit name? Also, the `config` is using a Bool in the fst part of the tuple, should we do the same here?

Perhaps we could use a datatype to remove boolean blindness.